src/test/groovy/com/cloudbees/groovy/cps/CpsTransformerTest.groovy

```diff
@@ -556,4 +556,10 @@ class CpsTransformerTest extends AbstractGroovyCpsTest {
     void varArgs() {
         assert evalCPS('def fn(String... args) { args.size() }; fn("one string")') == 1;
     }
+
+    @Issue("JENKINS-28277")
+    @Test
+    void currying() {
+        assert evalCPS('def nCopies = { int n, String str -> str*n }; def twice=nCopies.curry(2); twice("foo")') == "foofoo";
```

Comment:
You can only curry a `Closure`, so I assume you mean test that curries a non-CPS closure, which isn't very interesting IMO but I'm happy to add it. For the same reason, one cannot curry `DefaultGroovyMethods`.